scripts/switchtls-build.bash

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
instead of modifying the checked-in CMakePresets file, I think it would be better to mention `TLSUV_TLSLIB` and show how to set it in CMakeUserPresets.json.